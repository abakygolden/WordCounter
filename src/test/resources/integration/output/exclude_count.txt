Total count of excluded words encountered 72