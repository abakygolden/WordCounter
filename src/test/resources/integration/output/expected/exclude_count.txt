Total count of excluded words encountered 76